pkg/apis/serving/v1alpha1/kfservice_framework.go

```diff
@@ -29,9 +29,9 @@ type FrameworkHandler interface {
 
 const (
 	// ExactlyOneModelSpecViolatedError is a known error message
-	ExactlyOneModelSpecViolatedError = "Exactly one of [Custom, Tensorflow, SKLearn, XGBoost] must be specified in ModelSpec"
+	ExactlyOneModelSpecViolatedError = "Exactly one of [Custom, Tensorflow, TensorRT, SKLearn, XGBoost] must be specified in ModelSpec"
 	// AtLeastOneModelSpecViolatedError is a known error message
```

Comment:
Same as above.....TensorRT is a specialized inference server, rather than a model 'type'.....